modules/vault-elb/variables.tf

```diff
@@ -25,6 +25,12 @@ variable "vault_asg_name" {
 # These parameters have reasonable defaults.
 # ---------------------------------------------------------------------------------------------------------------------
 
+variable "access_logs" {
+  description = "Map containing access logging configuration for load balancer."
+  type        = map(string)
```

Comment:
This can't be a `map(string)`, because then all values must be `string`, whereas the [`access_logs` block](https://registry.terraform.io/providers/hashicorp/aws/latest/docs/resources/elb) can contain a `bool` value (`enabled`) and an `int` value (`interval`). Due to limitations in Terraform's type system, that meets we lamely have to fall back to `any`. I'd recommend explaining this in a comment too. Also, you'll want to set the default to `null`.